src/client/visualizations/heat-map/labelled-heatmap.tsx

```diff
@@ -59,18 +60,16 @@ export class LabelledHeatmap extends React.PureComponent<Props, State> {
     const { dataset } = this.props;
     const { hoveredRectangle } = this.state;
 
-    const [measure] = this.props.essence.getEffectiveSelectedMeasures().toArray();
-    const [firstSplit, secondSplit] = this.props.essence.splits.splits.toArray();
+    const measure = this.props.essence.getEffectiveSelectedMeasures().first();
+    const [firstSplit, secondSplit] = this.props.essence.splits.splits.slice(0, 2).toArray();
```

Comment:
You insist on keeping this in one line? :)

Reply:
But I want to be cool and use ES6 :(